src/cls/rgw/cls_rgw.cc

```diff
@@ -707,10 +707,27 @@ int rgw_bucket_prepare_op(cls_method_context_t hctx, bufferlist *in, bufferlist
   return write_bucket_header(hctx, &header);
 }
 
+static bool obj_name_filter(string& name)
```

Comment:
indentation looks wrong in function

Reply:
The format problem has been corrected

Reply:
add  tracker issue : http://tracker.ceph.com/issues/21565

Reply:
Function name should be something more descriptive.  "is_multipart_objname" or some such.
name can be const.  And this could be inline.